docs: add known issue for I/O stuck on ARM platforms (#1072)
harvester/docs: docs/advanced/longhorn-v2.md

  Volumes and images created using the new StorageClass are backed by the Longhorn V2 Data Engine.

## Known Issues

### I/O Operations May Become Stuck on ARM Platforms

> On ARM platforms, Longhorn V2 disks that use the SPDK NVMe bdev driver may experience stuck I/O operations. For more information, see [harvester/harvester#10710](https://github.com/harvester/harvester/issues/10710).

Reviewer: ```suggestion
On ARM64 platforms, Longhorn V2 disks that use the SPDK NVMe bdev driver can experience stalled I/O operations. For more information, see issue [#10710](https://github.com/harvester/harvester/issues/10710).
```

To work around the issue, perform the following steps:

1. [Remove the related Longhorn V2 disk](../host/host.md#remove-disks).

> 1. [Add the Longhorn V2 disk again](../host/host.md#add-additional-disks).

Reviewer: ```suggestion
1. [Remove the affected Longhorn V2 disk](../host/host.md#remove-disks).

1. [Add the Longhorn V2 disk again](../host/host.md#add-additional-disks).

1. Verify that the workaround was successful.

    ```
    kubectl -n longhorn-system get blockdevices.harvesterhci.io -o yaml
    ```

    The matching `BlockDevice` custom resource should have the following values:

    ```
    spec:
      provisioner:
        longhorn:
          diskDriver: aio
          engineVersion: LonghornV2
    status:
      provisionPhase: Provisioned
      state: Active
    ```
```

Starting with Harvester v1.9.0, the default disk driver for Longhorn V2 disks is `aio`. Re-adding the disk allows Harvester to use the default driver and avoid the I/O issue on ARM platforms. For more information, see [harvester/harvester#11041](https://github.com/harvester/harvester/issues/11041).

To verify that the workaround was successful, run the following command:

```
kubectl -n longhorn-system get blockdevices.harvesterhci.io -o yaml
```

The matching `BlockDevice` custom resource should have the following values:

```
spec:
  provisioner:
    longhorn:
      diskDriver: aio
      engineVersion: LonghornV2
status:
  provisionPhase: Provisioned
  state: Active

> ```

Reviewer: ```suggestion
:::note

Starting with Harvester v1.9.0, the default disk driver for Longhorn V2 volumes is changed to `AIO`. Removing and then re-adding the affected disk forces Harvester to use this new default driver and resolves the stalled I/O operations on ARM64 platforms.

:::
```
